src/cli.rs

```diff
@@ -227,3 +246,25 @@ pub struct IdentifyArgs {
     #[arg()]
     pub peer_addr: String,
 }
+
+#[derive(Args, Debug, Clone)]
+pub struct AnchorRequestArgs {
+    /// CAS API URL
+    #[arg(long)]
+    pub url: String,
+    /// Auth controller.
+    #[arg(long)]
+    pub controller: String,
+    /// Auth private key.
+    #[arg(long)]
+    pub private_key: String,
```

Comment:
Should we make this one arg at least work via ENV var and not the CLI so we do not encourage putting the private_key in the shell history?